recipes/apache-ant/bld.bat

```diff
@@ -0,0 +1,17 @@
+set JAVA_HOME=%LIBRARY_PREFIX%
```

Comment:
Can't we just set the `ANT_HOME` variable also to `%LIBRARY_PREFIX%`, so we don't have to copy the files after building?

Reply:
`JAVA_HOME` should be set automatically by the `openjdk` package, so there should be no need for it.

Reply:
Ok, good to know. But I meant that if we set the `ANT_HOME` variable and run `build.bat install`, it will install the files to the prefix, right?

Reply:
> `JAVA_HOME` should be set automatically by the `openjdk` package, so there should be no need for it.

Implemented changes.

Reply:
> Ok, good to know. But I meant that if we set the `ANT_HOME` variable and run `build.bat install`, it will install the files to the prefix, right?

Looks like **./build.sh install** cannot run on Linux and OSX due to lack of execute permissions when ANT_HOME is set to $PREFIX